app/api/entities/patient.rb

```diff
@@ -0,0 +1,16 @@
+module Entities
+  class Patient < Grape::Entity
+    expose :id
+    expose :address
+    expose :date_of_birth
+    expose :email
+    expose :full_name
+    expose :notes
+    expose :telephone_number
+    expose :passport_id
+    expose :allergies do
+      expose :allergies, using: API::Entities::Allergy
+      expose :allergies_additional
```

Comment:
what's the difference between `allergies` and `allergies_additional`? names should reflect the purposes of this or that actions. So, fir me the difference is not clear

Reply:
`allergies_additional` is a column of the table "patients" with text type data (to store more allergy information for a particular patient). `allergies` shows records from associated table "patient-allergies". In sum, these fields provide all information about allergies, so they are combined in the field `allergies`.